avocado/plugins/runners/podman_image.py

```diff
@@ -0,0 +1,71 @@
+import asyncio
+import time
+from multiprocessing import Process, SimpleQueue
+
+from avocado.core.nrunner.app import BaseRunnerApp
+from avocado.core.nrunner.runner import RUNNER_RUN_STATUS_INTERVAL, BaseRunner
+from avocado.core.utils import messages
+from avocado.utils.podman import Podman, PodmanException
+
+
+class PodmanImageRunner(BaseRunner):
+    """Runner for dependencies of type podman-image
+
+    This runner handles download and verification.
+
+    Runnable attributes usage:
+
+     * kind: 'podman-image'
+
+     * uri: the name of the image
+
+     * args: not used
+    """
+
+    name = 'podman-image'
+    description = f'Runner for dependencies of type {name}'
+
+    def _run_podman_pull(self, uri, queue):
+        try:
+            podman = Podman()
+            loop = asyncio.get_event_loop()
+            loop.run_until_complete(podman.execute('pull', uri))
+            result = 'pass'
+        except PodmanException:
+            result = 'fail'
```

Comment:
IMO it would be good to have information about failure. I would send an `StderrMessage` like in [avocado-instrumented runner](https://github.com/avocado-framework/avocado/blob/master/avocado/plugins/runners/avocado_instrumented.py#L108)